cloudesire-api-client/src/main/java/com/cloudesire/platform/apiclient/api/UserCompanyApi.java

```diff
@@ -14,6 +14,13 @@
 
 public interface UserCompanyApi
 {
+    @POST( "userCompany" )
+    Call<UserCompanyDTO> create( @Body UserCompanyDTO company );
+
+    /**
+     * @deprecated by {@link #create(UserCompanyDTO)}
+     */
+    @Deprecated
     @POST( "userCompany" )
     Call<UserCompanyDTO> create( @Body MinimalCompanyDTO company );
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS1133)